lib/cdn-site-hosting/cdn-site-hosting-construct.ts

```diff
@@ -77,6 +77,13 @@ export class CdnSiteHostingConstruct extends cdk.Construct {
             behaviors: [{ isDefaultBehavior: true }],
           },
         ],
+        errorConfigurations: [
+          {
+            errorCode: 404,
+            responseCode: 200,
+            responsePagePath: "/index.html",
+          },
+        ],
```

Comment:
Not sure when you wouldn't want this enabled for an SPA with routing, but might be overkill for something that doesn't have routing, so maybe it should be toggleable through a new option in `CdnSiteHostingConstructProps`?

Reply:
That suggestion sounds like the way to go - this can currently be used for static site hosting, too.

Reply:
I'm ultimately wondering if we just have an `isRoutedSpa` flag that controls both this and the error page in S3. I don't think we'll need the error page in S3 if this is configured.

Reply:
that sounds good to me, it makes sense to just have a simple flag for the calling app and have this library apply what we think are the best settings. I'll update the PR to do that.